### PR TITLE
added esConsumes to modules in RecoVertex/BeamSpotProducer

### PR DESCRIPTION
**Pull request by JamminJones**
#### PR description:

added esConsumes to 3 modules
#### PR validation:

the code compiles

### CODE REVIEW

**Comment by Dr15Jones** [comment]
The change was made under my supervision.

**Comment by Dr15Jones** [comment]
please test

File: RecoVertex/BeamSpotProducer/interface/BeamSpotOnlineProducer.h [hunk 1]
```diff
@@ -17,6 +17,10 @@ ________________________________________________________________**/
 #include "DataFormats/Common/interface/Handle.h"
 #include "FWCore/Framework/interface/ESHandle.h"
 #include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Utilities/interface/ESGetToken.h"
+
+#include "CondFormats/DataRecord/interface/BeamSpotObjectsRcd.h"
```
Comment (perrotta) [inline RecoVertex/BeamSpotProducer/interface/BeamSpotOnlineProducer.h:22]: These two includes can be removed from the implementation file, then

**Comment by Dr15Jones** [comment]
please test

**Comment by perrotta** [comment]
+1
- Technical
- Jenkins tests pass

**Comment by christopheralanwest** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**